src/io/bed.rs

```diff
@@ -323,7 +316,7 @@ where
 ///     &vec![808, 52, 109],
 ///     &vec![0, 864, 984],
 ///     ReqStrand::Reverse,
-/// )?;
+/// ).expect("");
```

Comment:
Suggestion for a message (based on the function content [here](https://github.com/rust-bio/rust-bio-types/blob/c2206e33e2261d7598ee48a5b18f9d54f08685d2/src/annot/spliced.rs#L255-L296)):
```suggestion
/// ).expect("Encountered a bio_types::annot::spliced::SplicingError.");
```